bundles/org.openhab.binding.ondilo/README.md

```diff
@@ -0,0 +1,122 @@
+# Ondilo Binding
+
+This binding integrates Ondilo ICO pool monitoring devices with openHAB, allowing you to monitor and automate your pool environment using openHAB’s rules and UI.
+
+## Supported Things
+
+`bridge:` Represents your Ondilo account (OAuth2)
+
+`ondilo:` Represents an individual Ondilo ICO device
+
+Ondilo ICO Pool as well as Spa devices are supported.
+Chlor as well as salt water.
+
+## Discovery
+
+Ondilo ICOs are discovered automatically after the bridge is authorized and online.
+Each Ondilo ICO will appear as a new Thing in the inbox.
+
+## Thing Configuration
+
+### `bridge` Thing Configuration
+
+- **url**: The URL of the openHAB instance. Required for the redirect during OAuth2 flow (e.g. `http://localhost:8080`)
+- **refreshInterval**: Polling interval of the bridge (i.e. config changes) (default: `1800 s`)
+
+### `ondilo` Thing Configuration
+
+- **id**: The Id of an Ondilo ICO device. Set via discovery service (e.g. `12345`)
+- **refreshInterval**: Polling interval of the device (i.e. measures) (default: `600 s`)
+
+Ondilo ICO takes measures every hour.
+Higher polling will not increase the update interval.

```

Comment:
Is it predicatable when the measure is taking place? In that case a cron scheduler or other mechanism could perform a poll just after the physicall device is updated. keeping the value in openHAB as up to date as possible.

Reply:
1 measure every hour. A poll can be performed, i.e. 61 min after the last measure.
I think I will keep cyclic polling, but trying to sync the schedule with the predicted measure time.

Reply:
This is now implemented with the latest changes